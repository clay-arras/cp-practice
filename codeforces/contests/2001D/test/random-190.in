1
10
14 25 1 5 9 3 17 7 17 17
