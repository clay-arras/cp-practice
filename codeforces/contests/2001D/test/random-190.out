8
14 25 1 5 9 3 17 7
